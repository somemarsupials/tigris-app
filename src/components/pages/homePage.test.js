import React from 'react';
import { shallow } from 'enzyme';
import HomePage from './homePage';

describe('HomePage', () => {
  it('matches snapshot', () => {
    expect(shallow(<HomePage />)).toMatchSnapshot();
  });
});
