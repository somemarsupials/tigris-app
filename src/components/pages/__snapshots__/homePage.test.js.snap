// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage matches snapshot 1`] = `
<div
  className="home"
>
  <h1>
    TIGRIS
  </h1>
</div>
`;
